Make the package autoreconfigurable

Adjust a minor detail in configure.ac in order to make the package
compatible with the autoconf/automake versions we are using in
Buildroot.

Signed-off-by: Benoît Thébaudeau <benoit.thebaudeau.dev@gmail.com>

Index: b/configure.ac
===================================================================
--- a/configure.ac
+++ b/configure.ac
@@ -1,6 +1,6 @@
 AC_INIT([logrotate],[3.8.9])
 
-AM_INIT_AUTOMAKE
+AM_INIT_AUTOMAKE([foreign])
 AC_DEFINE(_GNU_SOURCE)
 
 AM_EXTRA_RECURSIVE_TARGETS([test])
